Core/Config.cpp

```diff
@@ -862,6 +862,8 @@ static ConfigSetting graphicsSettings[] = {
 	ReportedConfigSetting("AutoFrameSkip", &g_Config.bAutoFrameSkip, false, true, true),
 	ConfigSetting("FrameRate", &g_Config.iFpsLimit1, 0, true, true),
 	ConfigSetting("FrameRate2", &g_Config.iFpsLimit2, -1, true, true),
+	ConfigSetting("AnalogFrameRate", &g_Config.iAnalogFpsLimit, 240, true, true),
```

Comment:
Should this belong to controls section? Related to #15406.

Reply:
This isn't the button or anything, it's the limit.  It's kinda more related to graphics.

-[Unknown]

Reply:
I'm just wondering if we want to allow different control profile to have different limit or not. Not really interested in the semantic meaning here :)

Not a big deal anyway